### PR TITLE
Add details about the new geomap widget

### PR DESCRIPTION
**Pull request by MirandaKapin**
### Preview
https://docs-staging.datadoghq.com/MirandaKapin/geomap/dashboards/widgets/geomap/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/dashboards/widgets/geomap.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Geomap Widget
+kind: documentation
+widget_type: "geomap"
```
Comment (apigirl) [inline content/en/dashboards/widgets/geomap.md:4]: this has to map to the api spec - https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/dashboard.yaml - which doesn't look like it includes this one yet. Do you know if there's a plan to add it?

File: content/en/dashboards/widgets/geomap.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  text: "Screenboards"
+- link: "/dashboards/graphing_json/"
+  tag: "Documentation"
+  text: "Building Dashboard using JSON"
+- link: "/notebooks/"
+  tag: "Documentation"
+  text: "Notebooks"
+---
+
+The geomap widget graphs any metric that has a across country tag/facet.
```
Comment (apigirl) [inline content/en/dashboards/widgets/geomap.md:22]:
```suggestion
The geomap widget graphs any metric that has an across country tag or facet.
```

File: content/en/dashboards/widgets/geomap.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+## Setup
+
+{{< img src="dashboards/widgets/geomap/geomap_setup.png" alt="Top List"  style="width:80%;">}}
+
+### Configuration
+
+1. Choose the data to graph:
+    * RUM: See [the RUM documentation][2] to configure a RUM query. 
+    * Log Events: See [the log search documentation][3] to configure a log event query.
+      * Note: The group by tag must include a country ISO Code. You can use the [GeoIP Processor][6] to do this, or manually include the [tags on ingest][8].
```
Comment (apigirl) [inline content/en/dashboards/widgets/geomap.md:35]:
```suggestion
      * **Note**: The group by tag must include a country ISO Code. You can use the [GeoIP Processor][6] to do this, or manually include the [tags on ingest][8].
```

File: content/en/dashboards/widgets/geomap.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    * Metric: See the [querying][1] documentation to configure a metric query. 
+      * Note: The group by tag must include a country ISO Code. You can [Generate Metrics from Injested Logs][7], or manually include the [tags on ingest][8].
```
Comment (apigirl) [inline content/en/dashboards/widgets/geomap.md:37]:
```suggestion
      * **Note**: The group by tag must include a country ISO Code. You can [generate metrics from ingested logs][7], or manually include the [tags on ingest][8].
```

File: content/en/dashboards/widgets/geomap.md (hunk 1 above)
Comment (MirandaKapin) [reply content/en/dashboards/widgets/geomap.md:4]: I'm pinging the team to add it

Comment (apigirl) [inline content/en/dashboards/widgets/geomap.md:4]:
```suggestion
```

File: content/en/dashboards/widgets/geomap.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+Optionally define its size and alignment.
+
+## API
+
+This widget can be used with the **Dashboards API**. Refer to the [Dashboards API][4] documentation for additional reference.
+
+The dedicated [widget JSON schema definition][5] for the top list widget is:
+
+{{< dashboards-widgets-api >}}
```
Comment (apigirl) [inline content/en/dashboards/widgets/geomap.md:61]:
```suggestion
```

**Review by apigirl — COMMENTED** [review]
removing API section

**Review by apigirl — APPROVED** [review]

**Pull request merged**